### ISSUE TITLE
Clarification of facets for staff profiles

### ISSUE THREAD

**Issue by sew268** [cul-it/uls#981]
Apologies if this is covered in a staff profiles issue already, but we need staff profiles with filters like we are planning on for Mann: department, expertise, etc.

**Comment by cappadona** [comment]
No worries, @sew268. I believe this is already covered by our initial implementation of the staff profiles vue app, which you can see in action on the [Mann dev site](https://dev-uls-mann-library-cornell-edu.pantheonsite.io/staff-profile/).

As mentioned previously, that page is pulling from content on the live central WP instance, which is currently (intentionally) not synced with the Mann dev instance where you have been managing staff content. The filters/facets will automatically adjust to the fields that are populated for Mann staff once the sync is re-established. I'm including a screenshot to illustrate how the data you've been entering from the Mann dev site would render.

Please let me know if this doesn't meet your needs or expectations.

![Screenshot_2021-02-22 cul-uws-vue](https://user-images.githubusercontent.com/409842/108738169-5667ea80-7501-11eb-905e-1f52621886dc.png)

**Comment by sew268** [comment]
@cappadona got it thanks!  And I just to verify that this would all occur on the Olin/Uris site as well - nothing additional would need to be done for Olin/Uris once this is in place and ready to go for Mann, correct?

**Comment by cappadona** [comment]
@sew268 Correct. The `[staff]` shortcode automatically prefilters staff for the current unit, so all that needs to be done for Olin/Uris is:
* manage staff content via the staff profiles content type in Olin/Uris admin
* add the `[staff]` shortcode to the applicable page (currently the [staff directory page on the Olin/Uris dev site](https://dev-uls-olinuris-library-cornell-edu.pantheonsite.io/staff-directory/) is still using the old custom loop variant)

**Comment by cappadona** [comment]
@sew268 Just checking in to ensure all your questions were answered and see whether we can close this issue.

**Comment by cappadona** [comment]
@sew268 Friendly nudge 😉

**Comment by sew268** [comment]
@cappadona yes and thanks!  Can you verify whether Olin/Uris staff will need to be reentered?  Just want to let Jessa know if that will e coming her way prior to launch :)

**Comment by cappadona** [comment]
Thanks @sew268. No need for Jessa to re-enter staff if she's happy with the metadata for the 43 that exist today. We still need to add the shortcode to the staff directory as I [mentioned above](https://github.com/cul-it/uls/issues/981#issuecomment-783509707).

**Comment by sew268** [comment]
@cappadona excellent - thanks!
